core/src/elo-connector/elo-connector.test-data.ts

```diff
@@ -0,0 +1,122 @@
+import type {MatchSummary, SeriesStatsPayload} from "./elo.types";
```

Comment:
Delete test data before merging?

Reply:
I believe this is used for the unit tests; which means it should remain

Reply:
Test data stays.